language-design.md

```diff
@@ -18,6 +18,53 @@ For each quote, describe what it was about the quote that led you pick it.
 **Response**
 
 
+1. "Many brilliant and talented people have contributed to R over the years, many
+of whom are renowned computer scientists for whom I have tremendous respect!
+But a computer scientist is not the same as a software engineer." [Smith, 2016]
+
+We picked this quote because while most of what we have been reading has been 
+from professors and researchers who study the design and implementation of 
+languages, we rarely get a voice from an "outsider" in this respect - someone 
+who just wants to use a language for a purpose and not to analyze its history 
+and design decisions. We think the goal of a well-designed and "good" DSL is to 
+be a tool that accomplishes the domain-specific goals easily or intuitively. 
+Smith does not think R is not well-designed, and traces the cause to the lack 
+of software engineers contributing to R.
```

Comment:
This is a very interesting viewpoint from someone strictly from industry and not academia! But I wonder who R is designed for. If R was tailored to those in academia, then it makes some sense for those in industry to say that R isn't well-designed.